hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/XMLUtils.java

```diff
@@ -61,4 +67,45 @@ public static void transform(
     // and send the output to a Result object.
     transformer.transform(new StreamSource(xml), new StreamResult(out));
   }
+
+  public static DocumentBuilderFactory newSecureDocumentBuilderFactory()
```

Comment:
javadocs, including SHOULD BE way to create new xml parsers

Reply:
added javadocs